### PR TITLE
Update changelog scripts to work with Python3

### PR DESCRIPTION
**Pull request by nywilken**
Now that Python 2 has been removed from Macos 12.x and replaced with
Python3 I'm updating the simple tools to work with the Python3
interrupter.

### CODE REVIEW

File: scripts/prepare_changelog.sh [hunk 1]
```diff
@@ -49,7 +49,7 @@ get_prs(){
             echo "Skipping PR ${PR_NUM}: labeled as tech debt, docs or website. (waiting a second so we don't get rate-limited...)"
             continue
         fi
-        echo "$(cat pull.json | python -m json.tool | jq -r '.title') - [GH-${PR_NUM}](https://github.com/hashicorp/packer/pull/${PR_NUM})"
+        echo "$(cat pull.json | python3 -m json.tool | jq -r '.title') - [GH-${PR_NUM}](https://github.com/hashicorp/packer/pull/${PR_NUM})"
```
Comment (lbajolet-hashicorp) [inline scripts/prepare_changelog.sh:52]: Unrelated to the current changes, but the `cat pull.json | python3 -m json.tool` part can be replaced by `python3 -m json.tool <pull.json`

File: scripts/prepare_changelog.sh [hunk 2]
```diff
@@ -11,7 +11,7 @@ is_doc_or_tech_debt_pr(){
         echo "jq not found"
         return 1
     fi
-    out=$(cat pull.json | python -m json.tool \
+    out=$(cat pull.json | python3 -m json.tool \
```
Comment (lbajolet-hashicorp) [inline scripts/prepare_changelog.sh:14]: Same here, the cat is unnecessary

**Review by lbajolet-hashicorp — APPROVED** [review]
Safe for the two nitpicks I have, LGTM!

File: scripts/prepare_changelog.sh (hunk 1 above)
Comment (nywilken) [reply scripts/prepare_changelog.sh]: Good call. I merged the changes in.

**Pull request merged**